src/web/arr/trove/chart.arr

```diff
@@ -398,12 +406,8 @@ data DataSeries:
     method show-outliers(self, show):
       self.constr()(self.obj.{show-outliers: show})
     end,
-    method min(self, min):
-      self.constr()(self.obj.{min: some(min)})
-    end,
-    method max(self, max):
-      self.constr()(self.obj.{max: some(max)})
-    end
+    min: min-method,
```

Comment:
@schanzer @jpolitz As discussed earlier, this line (along with the next line) should be removed. The JS side also needs to be updated to extract information from the chart window object rather than data series.

Reply:
I submitted #371 to clarify my proposal. Please review and discuss.